packages/zk-certificates/contracts/ClaimrSBT.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+contract claimrSignedSBT is Ownable, ERC721 {
+    uint256 private _nextTokenId;
+    string private _baseTokenURI;
+    mapping(string => bool) private _usedTokens;
+    address private _signee;
+    constructor(string memory name, string memory symbol, string memory baseTokenURI_, address signee_) ERC721(name, symbol) Ownable() {
+        _baseTokenURI = baseTokenURI_;
+        _signee = signee_;
+    }
+    function mint(string memory token, bytes memory signature) public {
+        require(verify(token, signature, _signee), "NOT_SIGNED_BY_SIGNEE");
+        require(!_usedTokens[token], "TOKEN_USED");
+        _usedTokens[token] = true;
+        uint256 tokenId = _nextTokenId++;
+        _mint(msg.sender, tokenId);
+    }
+    function setSignee(address newSignee) public onlyOwner {
+        _signee = newSignee;
+    }
+    function signee() public view virtual returns (address) {
+        return _signee;
+    }
+    function setBaseURI(string calldata baseURI) external onlyOwner {
+        _baseTokenURI = baseURI;
+    }
+    function _baseURI() internal view override returns (string memory) {
+        return _baseTokenURI;
+    }
+    function tokenURI(
+        uint256 tokenId
+    ) public view virtual override returns (string memory) {
+        require(
+            tokenId < _nextTokenId,
+            "ERC721Metadata: URI query for nonexistent token"
+        );
+        return _baseTokenURI;
```

Comment:
Is it right, that each token has the same URI? Asking because the `base` part in  `_baseTokenURI` sounds like it would be extended by the tokenid to get the token specific metadata.